### PR TITLE
Release 1.6.1

### PR DESCRIPTION
**Pull request by jezdez**
This should only be merged once we've done the 1.6.1 release.
The last commit of the 1.6.x branch basically will be the commit to tag as 1.6.1.

This contains #516 to fix a compatibility issue.

See https://github.com/jazzband/roadies/issues/148 for more info.

### CODE REVIEW

**Comment by jezdez** [comment]
@odinuge Do you know why the tests are failing?

**Comment by odinuge** [comment]
Not sure why the `tests/test_app_config.py` test fails, need to check some more to be sure. I don't think it makes much sense to merge this into master, since there has happened quite a lot in the travis config since v1.6.0. Take a look at this PR: https://github.com/jazzband/django-push-notifications/pull/501.

The tests in v1.6.0 is also targeting django master branch, that is currently v3.0.0-alpha-0, so no tests will pass targeting that.

**Comment by jezdez** [comment]
Well, in that case I don't see a way forward for a 1.6.1 release if the tests don't pass. It's kind of awful to users that are not willing to make the jump to 2.0.0 yet (or can't) though.

File: tox.ini [hunk 1]
```diff
@@ -21,7 +21,7 @@ deps =
 	djangomaster: https://github.com/django/django/archive/master.tar.gz
     apns2>=0.3.0
```
Comment (odinuge) [inline tox.ini]: Looks like we have to limit the apns2 version here too

Comment (jezdez) [reply tox.ini]: Odd, that's not needed since it's already in `install_requires` and tox will build a sdist and install the package in a venv anyway.

**Comment by jezdez** [comment]
I see, those tests were cleaned up in 5954578417f459372383f6217525575f4ddc161e. I'll patch the tests.

**Comment by jezdez** [comment]
Meh, only the flake8 tests fail.

**Comment by jezdez** [comment]
I'll need to get back to this later today.

**Comment by odinuge** [comment]
Nice work! I think you can solve the flake8 issues by pinning the version, just like in: https://github.com/jazzband/django-push-notifications/pull/501/commits/7c5019cdf0bb2e8ccbfb2c4995ca4cf71196dd5f

Did you look at cherry-picking this one too: https://github.com/jazzband/django-push-notifications/pull/492, so that we get python 3.7 support?

**Comment by jezdez** [comment]
@odinuge Nah, Python 3.7 can wait for 2.0.0.

**Comment by jezdez** [comment]
[1.6.1](https://pypi.org/project/django-push-notifications/1.6.1/) is out.

**Comment by jezdez** [comment]
Merged manually in 7d6d0a74344e8230a062e4c0416cca8828a99ca8.

**Pull request closed without merging**